lib/elixir/test/elixir/code_normalizer/formatted_ast_test.exs

```diff
@@ -173,29 +173,30 @@ defmodule Code.Normalizer.FormatterASTTest do
 
   describe "charlists" do
     test "without escapes" do
-      assert_same ~S['']
-      assert_same ~S[' ']
-      assert_same ~S['foo']
+      assert_same ~S[~c""]
```

Comment:
These tests were misleading since what we are actually comparing is sigils.

Reply:
Also fixes these in the tests 🧘
<img width="901" alt="Screenshot 2024-09-20 at 14 28 16" src="https://github.com/user-attachments/assets/ae530b0c-d71a-4bac-acb2-5c54c2e8d112">